source/common/network/BUILD

```diff
@@ -262,6 +263,7 @@ envoy_cc_library(
         "socket_interface_impl.h",
         "win32_socket_handle_impl.h",
     ] + select({
+        "//bazel:android": [],
```

Comment:
Any idea why we don't need this for ios? e.g. `bazel:ios_arm64`

Reply:
Since the iOS CI does not complain about io_uring, I guess that iOS does not build //bazel:linux targets but Android does.